lib/FFI/Platypus/Memory.pm

```diff
@@ -12,14 +12,14 @@ use base qw( Exporter );
 =head1 SYNOPSIS
 
  use FFI::Platypus::Memory;
- 
+
```

Comment:
I realize this is uncommon style, but this space is intentional because some (probably broken) POD readers render the literal blocks separately if you do not indent the space between.  I can fix this later though.

Reply:
My editors automatically trim trailing spaces X_X